config/env_blank.json

```diff
@@ -38,5 +38,12 @@
     "user": "",
     "password": ""
   },
-  "APPEND_TABLE_NAMES": ["job_run", "data_source_status"]
+  "APPEND_TABLE_NAMES": [
+    "job_run", "data_source_status",
+    "mivideo_media_started_hourly", "mivideo_media_creation"
+  ],
+  "mivideo": {
+    "/*  'umich-its-tl-reports-prod.json' can be found at https://umich.app.box.com/file/641207013676": "*/",
+    "service_account_json_filename": "umich-its-tl-reports-prod.json"
```

Comment:
Even though Box is behind SSO, I feel like this is too much information to provide here. I would probably just explain in the configuration README table what values are expected here, linked to the Google documentation where appropriate, and make sure other team members are aware of where to find this file you reference.

Reply:
🤷

Reply:
✅Fixed.